DrDocket/Date.h

```diff
@@ -54,6 +58,7 @@ class Date {
     bool operator< (const Date &rhs);
     bool operator<= (const Date &rhs);
     Date operator+ (const unsigned int &rhs);    //increment Date by days
+    static Month parseMonth(int month);
```

Comment:
Does this need to be static?

Reply:
It doesn't need to, but I thought it was a clearer interface to the code.  I.e This function can be used without having to construct a Date instance first.